music.js

```diff
@@ -1,17 +1,28 @@
-// Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
-// and a 'songs' property that is an empty array to start
-function Artist() {
-
+function Artist(name) {
+  this.name = name;
+  this.songs = [];
 }
 
-// Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
-// and a 'playCount' property that is an empty array to start
-function Song() {
-
-  // When a new song is instantiated, you'll want to add that song to its artist's 'songs' array.
-  // You'll probably want to do that in here somewhere...
+function Song(title, artist) {
+  this.title = title;
+  this.artist = artist;
+  this.playCount = 0;
+  Artist.prototype.add = function(title) {
+    this.songs.push(title)
+  }
+  artist.add(title);
```

Comment:
You probably want to refer to `this.artist` here, not just `artist`. Both will work in this scenario, but since we already went to the trouble of creating a property for this purpose, we should probably use it. In general, in a constructor function when you accept arguments to set as properties on the object, you don't use the arguments after they've been set as properties. This way, you preserve a 'single source of truth', so you always know what the authoritative piece of data is.